src/resources/parser/glb-parser.js

```diff
@@ -1039,14 +1039,15 @@ var createMaterial = function (gltfMaterial, textures, disableFlipV) {
             extractTextureTransform(clearcoatTexture, material, ['clearCoat']);
         }
         if (ccData.hasOwnProperty('clearcoatRoughnessFactor')) {
-            material.clearCoatGlossiness = ccData.clearcoatRoughnessFactor;
+            material.clearCoatGlossiness = 1.0 - ccData.clearcoatRoughnessFactor;
         } else {
             material.clearCoatGlossiness = 0;
         }
         if (ccData.hasOwnProperty('clearcoatRoughnessTexture')) {
             var clearcoatRoughnessTexture = ccData.clearcoatRoughnessTexture;
             material.clearCoatGlossMap = textures[clearcoatRoughnessTexture.index];
             material.clearCoatGlossMapChannel = 'g';
+            material.hasRoughnessMap = true;
```

Comment:
It is probably `isRoughnessMap` based on conventions used in engine, also as it is same texture (not separate).

Reply:
makes sense - switched to is instead of has - thanks!